modules/update-preparing-conditional.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_content-type: PROCEDURE
+[id="update-preparing-conditional_{context}"]
+= Assessing the risk of conditional updates
+
+A conditional update is a supported update target that is available but not recommended due to a known risk that applies to your cluster.
+The Cluster Version Operator (CVO) periodically queries the OpenShift Update Service (OSUS) for the most recent data about update possibilities, and some potential update targets have conditional risks.
+
+These conditional risks are evaluated by the CVO, and if the risk is not applicable to the cluster then the update is available to users as a recommended update.
+If the risk is determined to be applicable, or if for some reason CVO cannot evaluate the risk, then the update is available to users as a conditional update.
```

Comment:
suggested text: `then the update target is available  to the cluster as a conditional update.`